imap_processing/tests/conftest.py

```diff
@@ -15,6 +19,119 @@ def _set_global_config(monkeypatch, tmp_path):
     )
 
 
+@pytest.fixture(scope="session")
+def monkeypatch_session():
+    from _pytest.monkeypatch import MonkeyPatch
+
+    m = MonkeyPatch()
+    yield m
+    m.undo()
+
+
 @pytest.fixture(scope="session")
 def imap_tests_path():
     return imap_module_directory / "tests"
+
+
+# Furnishing fixtures for testing kernels
+# ---------------------------------------
+@pytest.fixture(autouse=True)
+def _autoclear_spice():
+    """Automatically clears out all SPICE remnants after every single test to
+    prevent the kernel pool from interfering with future tests. Option autouse
+    ensures this is run after every test."""
+    yield
+    spice.kclear()
+
+
+@pytest.fixture(scope="session")
+def spice_test_data_path(imap_tests_path):
+    return imap_tests_path / "spice/test_data"
+
+
+@pytest.fixture()
+def furnish_test_lsk(spice_test_data_path):
+    """Furnishes (temporarily) the testing LSK"""
+    test_lsk = spice_test_data_path / "naif0012.tls"
+    spice.furnsh(test_lsk)
+    yield test_lsk
+    spice.kclear()
+
+
+@pytest.fixture()
+def furnish_sclk(spice_test_data_path):
```

Comment:
Looks like this fixture isn't used, do you want to use it here, planning on needing it later? (I assume it might come in future PR of `test_time`, but just wanted to call it out)

Reply:
I do think that it will be useful later, especially in `test_time`. Mostly wanted to put it in to help others with testing fixtures that they might need.